SU2_CFD/src/solvers/CTurbSASolver.cpp

```diff
@@ -1253,332 +1253,6 @@ void CTurbSASolver::BC_Inlet_Turbo(CGeometry *geometry, CSolver **solver_contain
 
 }
 
-void CTurbSASolver::BC_Interface_Boundary(CGeometry *geometry, CSolver **solver_container, CNumerics *numerics,
-                                          CConfig *config, unsigned short val_marker) {
-
-  //  unsigned long iVertex, iPoint, jPoint;
-  //  unsigned short iVar, iDim;
```

Comment:
A pro-move I learned from a dear colleague in a company I wont name here: throw an error in the function, and if no-one complains for long enough, delete the code 👍 ... this full commenting of the function content is kinda the same